validator/accounts/v2/accounts_exit.go

```diff
@@ -101,7 +106,42 @@ func ExitAccountsCli(cliCtx *cli.Context, r io.Reader) error {
 		return nil
 	}
 
-	log.WithField("publicKeys", allAccountStr).Info("Voluntary exit was successful")
+	dialOpts := client.ConstructDialOptions(
+		cmd.GrpcMaxCallRecvMsgSizeFlag.Value,
+		flags.CertFlag.Value,
+		strings.Split(flags.GrpcHeadersFlag.Value, ","),
+		flags.GrpcRetriesFlag.Value,
+		flags.GrpcRetryDelayFlag.Value,
+	)
+	if dialOpts == nil {
+		return errors.New("failed to construct dial options")
+	}
+	conn, err := grpc.DialContext(cliCtx.Context, cliCtx.String(flags.BeaconRPCProviderFlag.Name), dialOpts...)
+	if err != nil {
+		return errors.Wrapf(err, "could not dial endpoint %s", flags.BeaconRPCProviderFlag.Name)
+	}
+	validatorClient := ethpb.NewBeaconNodeValidatorClient(conn)
+
+	var rawNotExitedKeys [][]byte
+	for _, key := range rawPublicKeys {
+		if err := client.ProposeExit(cliCtx.Context, validatorClient, keymanager, key); err != nil {
+			rawNotExitedKeys = append(rawNotExitedKeys, key)
+			log.WithError(err).Errorf("voluntary exit failed for account %s", key)
```

Comment:
Display the formatted key.

Reply:
please display key as hex string, check proposer and attester .go for consistency